### ISSUE TITLE
ValidationException: name: This value should have a length greater than 5. Currently Kev

### ISSUE THREAD

**Issue by Miserlou** [capless/kev#26]
When following along with the documentation..

```python
In [6]: kevin = models.TestDocument(name='Kev',is_active=True,no_subscriptions=3,state='NC',gpa=3.25)

In [7]: kevin
Out[7]: <TestDocument: Kev:None >

In [8]: kevin.save()
---------------------------------------------------------------------------
ValidationException                       Traceback (most recent call last)
<ipython-input-8-e3970196d46e> in <module>()
----> 1 kevin.save()

/Users/rjones/Projects/RedString/env/lib/python2.7/site-packages/kev/document.pyc in save(self)
    114 
    115     def save(self):
--> 116         self._db.save(self)
    117 
    118 

/Users/rjones/Projects/RedString/env/lib/python2.7/site-packages/kev/backends/s3/db.pyc in save(self, doc_obj)
     39 
     40     def save(self,doc_obj):
---> 41         doc_obj, doc = self._save(doc_obj)
     42         self._db.Object(self.bucket, self.get_full_id(
     43             doc_obj.__class__,doc_obj._id)).put(

/Users/rjones/Projects/RedString/env/lib/python2.7/site-packages/kev/backends/__init__.pyc in _save(self, doc_obj)
     54         doc = doc_obj._data.copy()
     55         for key, prop in list(doc_obj._base_properties.items()):
---> 56             prop.validate(doc.get(key), key)
     57             raw_value = prop.get_python_value(doc.get(key))
     58             if prop.unique:

/Users/rjones/Projects/RedString/env/lib/python2.7/site-packages/valley/mixins.pyc in validate(self, value, key)
     27             value = self.get_default_value()
     28         for i in self.validators:
---> 29             i.validate(value, key)
     30 
     31     def get_default_value(self):

/Users/rjones/Projects/RedString/env/lib/python2.7/site-packages/valley/validators.pyc in validate(self, value, key)
    198                     '{0}: This value should '
    199                     'have a length greater than '
--> 200                     '{1}. Currently {2}'.format(key, self.length, value)
    201                 )
    202         except TypeError:

ValidationException: name: This value should have a length greater than 5. Currently Kev
```

Because in the example, name has a limit of 5. I think this is a documentation bug, it just needs to be 

`
kevin = models.TestDocument(name='Kevin',is_active=True,no_subscriptions=3,state='NC',gpa=3.25)
`

and the error should be changed to "greater than or equal to"

**Comment by bjinwright** [comment]
Good point

**Comment by bjinwright** [comment]
@Miserlou Can you share the Document (model) that you are using?

**Comment by bjinwright** [comment]
Fixed in c76447221323db2bb1e14ebea03e3912a5438240 and latest [valley](https://www.github.com/capless/valley) release
